e2e-test/helper.py

```diff
@@ -0,0 +1,165 @@
+
+import json
+import uuid
+import random
+import string
+import wds_client
+import requests
+import time
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+workspace_manager_url = ""
+rawls_url = ""
+leo_url = ""
+
+def setup(bee_name):
+# define major service endpoints based on bee name
+    global workspace_manager_url
+    workspace_manager_url = f"https://workspace.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"workspace_manager_url: {workspace_manager_url}")
+    global rawls_url
+    rawls_url = f"https://rawls.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"rawls_url: {rawls_url}")
+    global leo_url
+    leo_url = f"https://leonardo.{bee_name}.bee.envs-terra.bio"
+    logging.debug(f"leo_url: {leo_url}")
+    return workspace_manager_url, rawls_url, leo_url
+
+# CREATE WORKSPACE ACTION
+def create_workspace(cbas, billing_project_name, header):
+    # create a new workspace, need to have attributes or api call doesnt work
+    api_call2 = f"{rawls_url}/api/workspaces";
+    request_body= {
+      "namespace": billing_project_name, # Billing project name
+      "name": f"api-workspace-{''.join(random.choices(string.ascii_lowercase, k=5))}", # workspace name
+      "attributes": {}};
+
+    response = requests.post(url=api_call2, json=request_body, headers=header)
+    logging.debug(f"url: {api_call2}")
+    logging.debug(f"headers: {header}")
+    logging.debug(f"response: {response}")
+    logging.debug(f"request_body: {request_body}")
+    
+    #example json that is returned by request: 'attributes': {}, 'authorizationDomain': [], 'bucketName': '', 'createdBy': 'yulialovesterra@gmail.com', 'createdDate': '2023-08-03T20:10:59.116Z', 'googleProject': '', 'isLocked': False, 'lastModified': '2023-08-03T20:10:59.116Z', 'name': 'api-workspace-1', 'namespace': 'yuliadub-test2', 'workspaceId': 'ac466322-2325-4f57-895d-fdd6c3f8c7ad', 'workspaceType': 'mc', 'workspaceVersion': 'v2'}

```

Comment:
This is a helpful comment, but consider prettying up the formatting for easier consumption?